### PR TITLE
fix: click notice icon  warring

### PR DESCRIPTION
**Pull request by iclae**

### CODE REVIEW

File: src/components/NoticeIcon/index.tsx [hunk 1]
```diff
@@ -117,11 +117,13 @@ export default class NoticeIcon extends Component<NoticeIconProps> {
       },
     );
     return (
-      <Spin spinning={loading} delay={0}>
-        <Tabs className={styles.tabs} onChange={this.onTabChange}>
-          {panes}
-        </Tabs>
-      </Spin>
+      <Fragment>
```
Comment (chenshuai2144) [inline src/components/NoticeIcon/index.tsx]: ts 中直接用 <></>

Comment (afc163) [reply src/components/NoticeIcon/index.tsx]: 没看懂这个改动

Comment (iclae) [reply src/components/NoticeIcon/index.tsx]: 原本的写法，点开通知时会报错

Comment (afc163) [reply src/components/NoticeIcon/index.tsx]: 原因是啥，为啥包一层就好了。

Comment (chenshuai2144) [reply src/components/NoticeIcon/index.tsx]: 无法再现 你再现地址吗？

Comment (iclae) [reply src/components/NoticeIcon/index.tsx]: 从V2手动升到V4, 直接在rightContent avatar组件下添加NoticeView，之后点击通知icon就会报错， 我不知道别人有没有这种情况

Comment (chenshuai2144) [reply src/components/NoticeIcon/index.tsx]: 给我一下报错截图，我看一下

Comment (iclae) [reply src/components/NoticeIcon/index.tsx]: Warning: Received `false` for a non-boolean attribute `selectable`.

If you want to write it to the DOM, pass a string instead: selectable="false" or selectable={value.toString()}.

If you used to conditionally omit it with selectable={condition && value}, pass selectable={condition ? value : undefined} instead.
    in div (created by Context.Consumer)
    in Spin (created by NoticeIcon)
    in LazyRenderBox (created by PopupInner)
    in div (created by PopupInner)
    in PopupInner (created by Popup)
    in Align (created by Popup)
    in AnimateChild (created by Animate)
    in Animate (created by Popup)
    in div (created by Popup)
    in Popup (created by Trigger)
    in Portal (created by Trigger)
    in Trigger (created by Dropdown)
    in Dropdown (created by Context.Consumer)
    in Dropdown (created by HeaderDropdown)
    in HeaderDropdown (created by NoticeIcon)
    in NoticeIcon (created by GlobalHeaderRight)
    in GlobalHeaderRight (created by Connect(GlobalHeaderRight))
    in Connect(GlobalHeaderRight) (created by GlobalHeaderRight)
    in div (created by GlobalHeaderRight)
    in GlobalHeaderRight (created by Connect(GlobalHeaderRight))
    in Connect(GlobalHeaderRight) (created by GlobalHeader)
............


Warning: React does not recognize the `expandIcon` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `expandicon` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in div (created by Context.Consumer)
    in Spin (created by NoticeIcon)
    in LazyRenderBox (created by PopupInner)
    in div (created by PopupInner)
    in PopupInner (created by Popup)
    in Align (created by Popup)
    in AnimateChild (created by Animate)
    in Animate (created by Popup)
    in div (created by Popup)
    in Popup (created by Trigger)
    in Portal (created by Trigger)
    in Trigger (created by Dropdown)
    in Dropdown (created by Context.Consumer)
    in Dropdown (created by HeaderDropdown)
    in HeaderDropdown (created by NoticeIcon)
    in NoticeIcon (created by GlobalHeaderRight)
    in GlobalHeaderRight (created by Connect(GlobalHeaderRight))
    in Connect(GlobalHeaderRight) (created by GlobalHeaderRight)
    in div (created by GlobalHeaderRight)
    in GlobalHeaderRight (created by Connect(GlobalHeaderRight))
    in Connect(GlobalHeaderRight) (created by GlobalHeader)

Comment (chenshuai2144) [reply src/components/NoticeIcon/index.tsx]: 这是传入了 html 不支持的属性造成的。和你的改动是无关的

**Comment by chenshuai2144** [comment]
https://github.com/ant-design/ant-design-pro/pull/4719

@iclae 定位到问题了

**Pull request closed without merging**